src/entries/popup/components/WarningInfo/WalletWipeWarningInfo.tsx

```diff
@@ -0,0 +1,159 @@
+import React, { useMemo, useState } from 'react';
+
+import { i18n } from '~/core/languages';
+import {
+  Box,
+  Button,
+  Column,
+  Columns,
+  Inline,
+  Row,
+  Rows,
+  Separator,
+  Stack,
+  Symbol,
+  Text,
+} from '~/design-system';
+import { Lens } from '~/design-system/components/Lens/Lens';
+
+import { Checkbox } from '../Checkbox/Checkbox';
+import {
+  IconAndCopyItem,
+  IconAndCopyList,
+} from '../IconAndCopyList.tsx/IconAndCopyList';
+
+const t = (s: string) =>
+  i18n.t(s, { scope: 'settings.privacy_and_security.wallets_and_keys' });
+
+interface WarningInfoProps {
+  iconAndCopyList: IconAndCopyItem[];
+  onProceed: () => void;
+  testId?: string;
+}
+
+export default function WarningInfo({
+  iconAndCopyList,
+  onProceed,
+  testId,
+}: WarningInfoProps) {
+  const [acknowledgeCheck, setacknowledgeCheck] = useState(false);
+
+  const buttonEnabled = useMemo(() => acknowledgeCheck, [acknowledgeCheck]);
```

Comment:
this useMemo does nothing
```suggestion
  const buttonEnabled = !!acknowledgeCheck;
```